package taskly.system.exception

/**
 * @author Alexandru Stoica
 * @version 1.0
 */

class TaskNotFoundException : RuntimeException("404 Task Not Found!")
